templates/emails/invite_user_msg.html

```diff
@@ -0,0 +1,13 @@
+<p>Hello {{ user.firstName }},</p>
+
+<p>You have been invited to the Dwellingly Platform!</p>
+
+<p>To finish creating your account, please reset your password here:</p>
```

Comment:
Added in those two details. The token is only set to 10 minutes at the moment. Would we ever want to widen that time window?

Reply:
Probably not. It is not ideal, or secure for that matter, to have a time window more than a few minutes. If the token happens to expire then they can request a new one by hitting the forgot password endpoint. It might be good to mention that in the email, but I think we can forgo that for now.